README.md

```diff
@@ -199,6 +199,87 @@ module.exports = {
 }
 ```
 
+<<<<<<< HEAD
```

Comment:
Looks from git rebase here :smile:

Reply:
Woops, sorry, should be good now? maybe?